package.json

```diff
@@ -4,7 +4,7 @@
   "description": "This repository contains code necessary to run the White House Office of Management and Budget (OMB) instance of [eRegulations](https://eregs.github.io) (a regulation parser, API, and viewer).",
   "main": "index.js",
   "scripts": {
-    "test": "jest",
+    "test": "jest --coverage",
```

Comment:
Do we need the flag here if `collectCoverage` is specified in the `jest` config?